### PR TITLE
Kraken Flash Crash Article

### PR DESCRIPTION
**Pull request by JediFaust**
Kraken Flash Crash of Bitcoin-Canadian Dollar pair

### CODE REVIEW

File: content/attacks/posts/2019-06-02-Kraken.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Attackers
+
+The identity of the attacker remains unknown.
+
+## Losses
+
+Kraken experienced losses of around $10,470,000 USD worth of 1,200 BTC.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-06-02-Kraken.md:22]: "...losses of 1,200 BTC, worth around $10,470,000 USD at the time of the incident"

File: content/attacks/posts/2019-06-02-Kraken.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Timeline
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-06-02-Kraken.md:24]: was there a response/comments from Kraken?

File: content/attacks/posts/2019-06-02-Kraken.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- **June 2, 2019:** The BTC/CAD pair on Kraken [experienced a flash crash](https://blockonomi.com/bitcoin-flash-crash-kraken/), dropping by over 99% in a matter of seconds.
+- **June 2, 2019, 03:02 PM:** An independent crypto expert [suggests details about the hacker's actions](https://twitter.com/Beetcoin/status/1135199936654565376).
+
+## Security Failure Causes
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-06-02-Kraken.md:29]: any info on how they gained access to the other account?

File: content/attacks/posts/2019-06-02-Kraken.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+loss: 10470000
+---
+
+## Summary
+
+On June 2, 2019, Kraken, a respected cryptocurrency exchange, experienced a flash crash on its Bitcoin-to-Canadian Dollar (BTC/CAD) trading pair. In a matter of seconds, the price of BTC dropped from $11,200 CAD to just over $101 CAD, a decline of over 99%. The incident resulted in losses of approximately $10,470,000 USD, equivalent to 1,200 BTC. Analysts suggest that this event was executed by a hacker attempting to siphon funds from Kraken. The attacker is alleged to have executed the flash crash using a specific method. Firstly, they managed to gain access to an account that held 1,200 BTC, but they were unable to withdraw the funds due to two-factor authentication. To maneuver around this, they set up their own buy orders, capping them at $100 CAD, on the BTC/CAD trading pair, which was not frequently traded. Then they sold off the entire 1,200 BTC into this illiquid market. As a result of this move, they effectively transferred the funds to themselves. Through this strategy, the attacker managed to become the legitimate owner of the 1,200 BTC, securing them at a price considerably below the prevailing market rates.
```
Comment (jhirschkorn) [inline content/attacks/posts/2019-06-02-Kraken.md:14]: Firstly -> First,

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2019-06-02-Kraken.md (hunk 2 above)
Comment (JediFaust) [reply content/attacks/posts/2019-06-02-Kraken.md:24]: They are [refused to accept responsibility](https://www.bnnbloomberg.ca/ether-s-50-selloff-was-genuine-not-an-error-kraken-ceo-says-1.1567379) on ETH flash crash on 2021, but nothing official, related to the current incident

File: content/attacks/posts/2019-06-02-Kraken.md (hunk 3 above)
Comment (JediFaust) [reply content/attacks/posts/2019-06-02-Kraken.md:29]: Nothing

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**